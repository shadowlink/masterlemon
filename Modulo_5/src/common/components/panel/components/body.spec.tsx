import * as React from 'react';
import { shallow } from 'enzyme';
import { Body } from './body';

describe('common/components/panel/body tests', () => {
  it('should render as expected when passing required properties', () => {
    // Arrange

    // Act
    const component = shallow(
      <Body>
        <h1>Test children component</h1>
      </Body>
    );

    // Assert
    expect(component).toMatchSnapshot();
  });
});
