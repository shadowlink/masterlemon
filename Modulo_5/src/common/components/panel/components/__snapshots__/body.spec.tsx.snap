// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/components/panel/body tests should render as expected when passing required properties 1`] = `
<ul
  className="list-group list-group-flush"
>
  <li
    className="list-group-item"
  >
    <h1>
      Test children component
    </h1>
  </li>
</ul>
`;
